pkg/minikube/proxy/cidir_test.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2019 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package proxy
+
+import (
+	"testing"
+)
+
+var bTests = []struct {
+	ip             string
+	block          string
+	expectedResult bool
+	expectedErr    error
+}{
+	{"", "192.168.0.1/32", false, nil},
+	{"192.168.0.1", "192.168.0.1/32", true, nil},
+	{"192.168.0.2", "192.168.0.1/32", false, nil},
+	{"192.168.0.1", "192.168.0.1/18", true, nil},
+	{"abcd", "192.168.0.1/18", false, nil},
+}
+
+func TestIsInBlock(t *testing.T) {
+
```

Comment:
Move bTests into this function, as readers of a test shouldn't have to bounce around to understand what is being tested.

Also, use subtests (t.Run): https://blog.golang.org/subtests

I recommend adding a "name" field to your tests table to be passed into t.Run(), so that users understand what the intent of the subtest is.